modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/shared/ClaySpinner.es.js

```diff
@@ -0,0 +1,33 @@
+import React, {Component} from 'react';
+import {PropTypes} from 'prop-types';
+import getCN from 'classnames';
+
+class ClaySpinner extends Component {
+	static propTypes = {
+		light: PropTypes.bool,
+		loading: PropTypes.bool,
+		size: PropTypes.oneOf(['sm'])
+	};
+
+	render() {
+		const {light, loading, size} = this.props;
+
+		const classes = getCN(
+			'loading-animation',
+			{
+				'loading-animation-light': light,
+				'loading-animation-sm': size === 'sm'
+			}
+		);
+
+		return loading && (
+			<span
+				aria-hidden="true"
```

Comment:
Should this be here?

Reply:
This was in the markup in https://clayui.com/docs/components/loading-indicator.html. Since there is no content in them, should it just stay ignored by screen readers?

Reply:
I think for clay they assume it will be hide/show for the indicator which is why they would label it as hidden. For this use case though, you are entirely omitting it from the markup so there should be no need to label it as hidden.

Reply:
This is a helpful reference, https://www.w3.org/WAI/PF/aria/states_and_properties#aria-hidden

Reply:
Ahhh that makes sense. I'll remove this line. 👍